### PR TITLE
Salesforce!

### PR DESCRIPTION
**Pull request by ebrodymoore**
Change over docs to salesforce

### CODE REVIEW

**Comment by ebrodymoore** [comment]
friendly reminder on this @juliasourceress @kpsource

**Comment by kpsource** [comment]
This looks good!

File: handbook/bizops/index.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ If you need clarification or are not sure about any of the above, ask a question
 We collect data from the following:
 
 * Google Analytics: Website analytics for Sourcegraph marketing and docs pages (not Sourcegraph.com)
-* HubSpot: Marketing automation and CRM
+* HubSpot: Marketing automation
+* Salesforce: Customer Management Relationship system (CRM)
```
Comment (dadlerj) [inline handbook/bizops/index.md:57]:
```suggestion
* Salesforce: Customer Relationship Management system (CRM)
```

File: handbook/communication/index.md [hunk 2]
```diff
@@ -16,7 +16,8 @@ These places are the source of truth for information at Sourcegraph. Information
 - [Project roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#)
 - [Issue tracker](https://github.com/sourcegraph/sourcegraph)
 - [productboard](https://sourcegraph.productboard.com/)
-- [HubSpot](https://app.hubspot.com)
+- [HubSpot](https://app.hubspot.com) (contacts only)
```
Comment (dadlerj) [inline handbook/communication/index.md:19]: What does "(contacts only)" mean?

File: handbook/sales/records.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # How to record emails, calls, and notes
 
-The [source of truth for our communications with customers is HubSpot](../communication.md). 
+The [source of truth for our communications with customers is Salesforce](../communication.md). 
 
-- All emails interacting with a customer must be logged to HubSpot (either by using the HubSpot Chrome extension to add a "Log" checkbox in Gmail, or by BCC'ing our HubSpot account's private ingestion email address (see the "HubSpot BCC email address" note in 1password).
-- Notes from all phone calls with customers should be logged in HubSpot. The call notes should be logged to the Deal, rather than the Contact or Customer (i.e., `/deal/` should appear in the URL). Click on the **Calls** tab at the top of the deal overview, and click the **Log Call** button.
+- All emails interacting with a customer must be logged to Salesforce (either by using the [Einstein Activity Capture](https://help.salesforce.com/articleView?id=einstein_sales_aac.htm&type=5) or manually to the relevant object.
```
Comment (dadlerj) [inline handbook/sales/records.md:5]:
```suggestion
- All emails interacting with a customer must be logged to Salesforce (either by using [Einstein Activity Capture](https://help.salesforce.com/articleView?id=einstein_sales_aac.htm&type=5 or by manually adding the email to the relevant object).
```

Comment (dadlerj) [reply handbook/sales/records.md:5]: This link isn't very helpful... I was hoping to go install it! :) What do I as a user need to do to get this?

File: handbook/sales/salesforce.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+TBD
+
+## HubSpot and Salesforce
+
+### HubSpot to Salesforce sync
+
+All HubSpot contacts are synced to Salesforce, usually within 15 minutes. The [complete settings are in HubSpot](https://app.hubspot.com/integrations-settings/2762526/installed/salesforce/contacts), but the most important of the synced fields are:
+- Basic contact info (name, email)
+- Their lead source based on First Page Seen
+- Most Zoominfo enriched fields
```
Comment (dadlerj) [inline handbook/sales/salesforce.md:50]:
```suggestion
- Most ZoomInfo enriched fields
```

File: handbook/bizops/index.md (hunk 1 above)
Comment (ebrodymoore) [reply handbook/bizops/index.md:57]: Ha 🤦‍♂️

File: handbook/communication/index.md (hunk 2 above)
Comment (ebrodymoore) [reply handbook/communication/index.md:19]: As opposed to deals and companies. Will make a clarification

**Pull request merged**